### PR TITLE
Add new CRIU API methods

### PR DESCRIPTION
**Pull request by tajila**
Add new CRIU API methods

Add capabilities to set tcpEstablished, trackMemory and autoDedup

Signed-off-by: Tobi Ajila <atobia@ca.ibm.com>

### CODE REVIEW

**Comment by tajila** [comment]
@DanHeidinga Please review these changes

File: jcl/src/openj9.criu/share/classes/org/eclipse/openj9/criu/CRIUSupport.java [hunk 1]
```diff
@@ -222,6 +228,42 @@ public CRIUSupport setFileLocks(boolean fileLocks) {
 		return this;
 	}
 
+	/**
+	 * Controls whether to dump file locks.
```
Comment (DanHeidinga) [inline jcl/src/openj9.criu/share/classes/org/eclipse/openj9/criu/CRIUSupport.java:232]: Comment looks like a copy/paste error.  Is this the intention:
```suggestion
	 * Controls whether to re-establish TCP connects
```

**Review by DanHeidinga — APPROVED** [review]
lgtm

**Comment by DanHeidinga** [comment]
Jenkins compile xlinux jdk11

**Comment by DanHeidinga** [comment]
A single compile should be sufficient to validate this

**Pull request merged**